onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -72,6 +72,7 @@ def inverse(g, self):
 
     _reg(inverse)
 
+    @torch.onnx.symbolic_helper.parse_args("v", "s")
```

Comment:
Matbe we should have a guideline on WHEN to use `parse_args`?

Reply:
Sure. What do you think?